src/cityreader/cityreader.py

```diff
@@ -1,41 +1,78 @@
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
+import csv
 
+# https://stackoverflow.com/questions/22282760/filenotfounderror-errno-2-no-such-file-or-directory
+# import os
+
+# cwd = os.getcwd()  # Get the current working directory (cwd)
+# files = os.listdir(cwd)  # Get all the files in that directory
+# print("Files in %r: %s" % (cwd, files))
+# test_stretch.py', 'cityreader.py', 'cities.csv', 'test_cityreader.py'
+
+
+class City:
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
+
+    def __str__(self):
+        return (f"{self.name}: {self.lat}, {self.lon}")
+
+
+# c = City("New York", 150, 160)
+# print(c)
 
 # We have a collection of US cities with population over 750,000 stored in the
 # file "cities.csv". (CSV stands for "comma-separated values".)
+
+
+# https://docs.python.org/3/library/csv.html
+
+
+# In the body of the `cityreader` function, use Python's built-in "csv" module
+# to read this file so that each record is imported into a City instance.
 #
-# In the body of the `cityreader` function, use Python's built-in "csv" module 
-# to read this file so that each record is imported into a City instance. Then
+# Then
 # return the list with all the City instances from the function.
 # Google "python 3 csv" for references and use your Google-fu for other examples.
-#
+
 # Store the instances in the "cities" list, below.
-#
+
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+
+
 cities = []
 
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    with open('cities.csv') as csvfile:
+        csv_reader = csv.reader(csvfile, delimiter=',')
+        next(csv_reader)
+        for row in csv_reader:
```

Comment:
The test for the STRETCH task didn't pass, but you still received a "2-star" rating for this sprint challenge, @MelvinPeepers ! My apologies for getting to this code review so late! 😅 
<img width="626" alt="melvin_stretch" src="https://user-images.githubusercontent.com/16110931/76147869-a0488e00-606e-11ea-9c6a-2634e2eb5c90.png">